src/solver/optimisation/opt_optimisation_lineaire.cpp

```diff
@@ -167,7 +167,8 @@ bool runWeeklyOptimization(const SingleOptimOptions& options,
             return false;
         }
 
-        if (problemeHebdo->ExportMPS != Data::mpsExportStatus::NO_EXPORT)
+        if (problemeHebdo->ExportMPS != Data::mpsExportStatus::NO_EXPORT
```

Comment:
Shouldn't there be a rule at a higher level which compute whether or not to export data like it is done here but in a more "centralized" way ?

Reply:
maybe just a boolean available in problem/study/options/whatever is available